src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,65 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.pagefactory.AjaxElementLocatorFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePage {
+    protected WebDriver driver;
+    protected WebDriverWait wait;
+    protected Actions actions;
+
+    @FindBy(css = ".songs")
+    WebElement allSongs;
+    @FindBy(css = ".fa-plus-circle")
+    WebElement addNewPlaylist;
+    @FindBy(css = "[data-testid='playlist-context-menu-create-simple']")
+    WebElement chooseCreateNewPlaylist;
```

Comment:
**Base Page \ Base Class** — Implements the necessary methods for work with webdriver.

The methods `addPlaylist()`,selectPlaylist(),  must be stored in the `class HomePage` because the locators are visible on the web site -home page

![image](https://user-images.githubusercontent.com/104761482/219305171-82698152-35b9-470d-9a11-2f008343410f.png)